source/_patterns/20-molecules/medium/medium.scss

```diff
@@ -93,6 +94,16 @@
   }
 }
 
+.medium--fullwidth-element.medium--fullwidth-element {
+  margin: 0;
+
+  .media-still,
```

Comment:
.media-still und .media-caption gehören eigentlich nicht in dieses file - eigene modifier?